### PR TITLE
[17.0][MIG] account_move_name_sequence: Migration to 17.0

### PR DESCRIPTION
**Pull request by moitabenfdz**

### CODE REVIEW

File: account_move_name_sequence/models/account_move.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            return False
+        return super(AccountMove, invoices_other_sequences)._is_end_of_seq_chain()
+
+    def _fetch_duplicate_supplier_reference(self, only_posted=False):
+        moves = self.filtered(lambda m: m.is_purchase_document() and m.ref)
+        if moves:
+            self.flush_model(["name", "journal_id", "move_type", "state"])
+        return super()._fetch_duplicate_supplier_reference(only_posted=only_posted)
+
+    def _get_last_sequence(self, relaxed=False, with_prefix=None):
```
Comment (hussain) [inline account_move_name_sequence/models/account_move.py:78]: `lock=True` is removed from ` _get_last_sequence` args, any reason for removal?

Comment (hussain) [reply account_move_name_sequence/models/account_move.py:78]: Nevermind, `lock=True` arg is removed in V17.0

**Review by hussain — APPROVED** [review]
Functional OK

**Comment by etobella** [comment]
Can you merge both migration commits and rebase :smile: ?

**Review by njeudy — APPROVED** [review]
ok for me tests in real database without problem, code review (as far as I can)

**Pull request merged**